unified-codes/apps/web/src/components/containers/explorer/ExplorerSearchBar.tsx

```diff
@@ -10,31 +10,40 @@ import { IState } from '../../../types';
 import { ITheme } from '../../../styles';
 
 const styles = (_: ITheme) => ({
-    input: { paddingLeft: 15 },
-    button: { marginTop: 15 }
+  input: { paddingLeft: 15 },
+  button: { marginTop: 15 },
 });
 
 const mapDispatchToProps = (dispatch: React.Dispatch<IExplorerAction>) => {
-    const onChange = (input: string) => dispatch(ExplorerActions.updateInput(input));
-
-    const onClear = () => batch(() => {
-        dispatch(ExplorerActions.resetInput());
-        dispatch(ExplorerActions.resetFilterBy());
-        dispatch(ExplorerActions.updateEntities());
+  const onChange = (input: string) => dispatch(ExplorerActions.updateInput(input));
+
+  const onClear = () =>
+    batch(() => {
+      dispatch(ExplorerActions.resetPage());
+      dispatch(ExplorerActions.resetInput());
+      dispatch(ExplorerActions.resetFilterBy());
+      dispatch(ExplorerActions.updateEntities());
     });
```

Comment:
Even though this is in a `batch`, it feels like `resetPage` makes more sense at the end rather than the beginning?